.travis.yml

```diff
@@ -1,18 +1,36 @@
 language: python
+
 python:
-    - "3.5"
-    - "3.6"
-    - "3.7-dev"
+  - "3.5"
+  - "3.6"
 
 os:
   - linux
 
+matrix:
+  fast_finish: true
+  include:
+  - python: 3.7
+    dist: xenial
+    sudo: true
+
 install:
   - pip3 install -r requirements.txt
-  - pip3 install bandit
+  - pip3 install bandit setuptools-scm
```

Comment:
Possibly add setuptools as well, because it doesn't actually (as far as I can tell) depend on setuptools, and you might get stuck with an older slightly non-compatible setuptools.